<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsCfg</class>
 <widget class="QDialog" name="SettingsCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="focustab_">
      <attribute name="title">
       <string>关注设定</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>指数设定</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="4">
           <widget class="QCheckBox" name="ft_A50">
            <property name="text">
             <string>A50期货</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="hk_hs_Index">
            <property name="text">
             <string>恒生指数</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="us_gold">
            <property name="text">
             <string>黄金期货</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QCheckBox" name="sh_hs300_Index">
            <property name="text">
             <string>沪深300</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="us_silver">
            <property name="text">
             <string>白银期货</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="sh_kc50_Index">
            <property name="text">
             <string>科创50</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="sh_sz_Index">
            <property name="text">
             <string>上证指数</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="sz_sz_Index">
            <property name="text">
             <string>深圳成指</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="sz_cyb_index">
            <property name="text">
             <string>创业板指</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="sz_zxb_Index">
            <property name="text">
             <string>中小板指</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QCheckBox" name="sh_sz50_Index">
            <property name="text">
             <string>上证50</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QCheckBox" name="sz_cyb50_index">
            <property name="text">
             <string>创业板50</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>自选股设定</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1" colspan="2">
           <widget class="QTableWidget" name="tableWidget">
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="InfoTips">
      <attribute name="title">
       <string>消息提示</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QCheckBox" name="info_rt">
         <property name="text">
          <string>实时信息</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="hq_zxg">
         <property name="text">
          <string>自选股行情</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="hq_index">
         <property name="text">
          <string>指数行情</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="northbound_rt">
         <property name="text">
          <string>北向实时资金</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="borthbound_top">
         <property name="text">
          <string>北向TOP10成交</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
